frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -332,3 +332,19 @@ Feature: Create the pipeline from builder page
               And user clicks on Create button
               And user clicks on Logs tab in PipelineRun details page
              Then user will be able to see the output in sum and multiply task
+
+
+        @regression @manual @odc-6377
+        Scenario: Disable Tektonhub integrations in the pipeline builder : P-02-TC22
+            Given user is at Search page
+              And user searches 'TektonConfig' in Resources dropdown
+              And user selects config with apiVersion operator.openshift.io/v1 option from Resources dropdown
+              And user clicks on "cluster" Name in Consoles
```

Comment:
The name of the tektonConfig CR will be `config`
![image](https://user-images.githubusercontent.com/9964343/146739348-34939f15-ad2b-45a2-ab99-3da7385c26ef.png)

Reply:
Updated